RocketChat/Rocket.Chat
# [IMPROVE] Rewrite remove room invite modal

## Proposed changes (including videos or screenshots)


### before
![image](https://user-images.githubusercontent.com/27704687/143086855-1904dbf3-fb32-4318-b744-95390d68ada2.png)

### after
![image](https://user-images.githubusercontent.com/27704687/143086365-d96fc9d2-12c6-4123-96a6-d8fd91cdf93a.png)



## Issue(s)


## Steps to test or reproduce

- First create a invite link: go to some room, click on members and invite link button
- Go to admin > Invites
- Click on the "X" button in some invite row to see the changes

## Further comments
